src/sentry/api/serializers/models/grouptombstone.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+from itertools import izip
+import six
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.constants import LOG_LEVELS
+from sentry.models import (
+    GroupTombstone, User
+)
+
+
+@register(GroupTombstone)
+class GroupTombstoneSerializer(Serializer):
+
+    def get_attrs(self, item_list, user):
+        project_set = set(i.project for i in item_list)
+        projects = {
+            p.id: d for p, d in izip(project_set, serialize(project_set, user))
```

Comment:
This isn't likely to happen, but since `set` doesn't guarantee ordering, zipping the `project_set` to the result of `serialize` isn't necessarily safe. It's safer to either:

- cast `project_set` to a list on L17 here, or
- replace `project_set` with a mapping and use `sentry.utils.functional.apply_keys`, e.g.

```python
projects = apply_values(
  lambda projects: serialize(projects, user),
  {i.project.id: i.project for in in item_list},
)
```